ios/brave-ios/Sources/BraveWallet/Crypto/WalletActions/DepositTokenView.swift

```diff
@@ -409,6 +436,17 @@ private struct DepositDetailsView: View {
       }
     }
   }
+
+  private func buildQRCodeViewModel(
+    account: BraveWallet.AccountInfo,
+    bitcoinAccounts: [String: BraveWallet.BitcoinAccountInfo]
+  ) -> QRCodeViewModel {
+    if let bitcoinAccount = bitcoinAccounts[account.accountId.uniqueKey] {
+      return .init(name: account.name, address: bitcoinAccount.nextReceiveAddress.addressString)
```

Comment:
Hmm, when `next_receive_address` updates, does [AccountsChanged](https://github.com/brave/brave-core/blob/master/components/brave_wallet/common/brave_wallet.mojom#L914-L915) fire? Uncommon case, but I think this address _could_ change with Deposit screen open for Bitcoin account, if so then we likely want to update this address & QR Code on Deposit. Might be issue on desktop too cc @supermassive

Reply:
Did we ever resolve this?

Reply:
I don't believe so. Desktop [polls the receive address every 10 minutes](https://github.com/brave/brave-core/blob/d8655cf7ac6851d55a5a4126966ed7297acc55dd/components/brave_wallet_ui/common/slices/api.slice.extra.ts#L269), though I'm not sure if there is a better way (IIRC we only monitor blockchain when there is a pending tx). @supermassive any thoughts?